src/rustc.rs

```diff
@@ -8,10 +8,11 @@ use crate::errors::*;
 use crate::extensions::CommandExt;
 
 pub struct TargetList {
-    triples: Vec<String>,
+    pub triples: Vec<String>,
 }
 
 impl TargetList {
+    // TODO remove pub
```

Comment:
Anything blocking this?

Reply:
made it pub for using it in config tests